__tests__/input.test.tsx

```diff
@@ -96,12 +97,15 @@ describe('Input component behaviour', () => {
             isEqual: jest.fn(_ => true),
         };
 
+        const props = { autofocusCandidate: autofocus };
+
         return shallowRender
             ? shallow(<Input id={propID} parentId={propparentId} flowKey={propflowKey} />)
-            : mount(<Input id={propID} parentId={propparentId} flowKey={propflowKey} />);
+            : mount(<Input id={propID} parentId={propparentId} flowKey={propflowKey} {...props} />);
     }
 
     afterEach(() => {
+        document.activeElement.blur();
```

Comment:
Why does this need to occur after every test case? Can it not just be applied to the applicable tests?

Reply:
If I cleanup the focus at the last line of some specific test and some assertion of that test fails, the focus will never be cleaned, and if there are more test after this one depending of a clean focus they would fail.

Anyways I will remove that line because I have found that the unmount method in the afterEach already clean the activeElement.